tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -96,6 +96,112 @@ Management network - valid
+
+    Log To Console  Pinging from bridge to public container.
+    ${id}=  Run  docker %{VCH-PARAMS} run -d busybox ping -c 30 ${ip}
+
+    Log To Console  Attach to running container.
+    ${out}=  Run  docker %{VCH-PARAMS} attach ${id}
+
+    Should Contain  ${out}  64 bytes from ${ip}
+    Log To Console  Ping test succeeded.
+
+    Cleanup VIC Appliance On Test Server
+
+Connectivity Bridge to Management
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${out}=  Run  govc host.portgroup.remove bridge
+    ${out}=  Run  govc host.portgroup.remove management
+
+    Log To Console  Create a bridge portgroup
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitch0 bridge
+
+    Log To Console  Create a management portgroup.
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitch0 management
+
+    ${output}=  Run  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} --force=true --bridge-network=bridge --compute-resource=%{TEST_RESOURCE} --container-network management --container-network vm-network --container-network-ip-range=management:10.10.10.0/24 --container-network-gateway=management:10.10.10.1/24 --no-tlsverify
+
+    Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}  ${true}
+    Log To Console  Installer completed successfully: %{VCH-NAME}
+
+    Log To Console  Creating management container
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --net=management --name m1 busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+
+    Log To Console  Starting management container
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start m1
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error:
+ 
+    Log To Console  Creating bridge container
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --net=bridge --name b1 busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+
+    Log To Console  Starting bridge container
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start b1
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error:
+
+    Log To Console  Getting IP for management container
+    ${ip}=  Run  docker %{VCH-PARAMS} inspect --format '{{range .NetworkSettings.Networks}}{{.IPAddress }}{{end}}' m1
+
+    Log To Console  Pinging from bridge to management container.
+    ${id}=  Run  docker %{VCH-PARAMS} run -d busybox ping -c 30 ${ip}
+
+    Log To Console  Attach to running container.
+    ${out}=  Run  docker %{VCH-PARAMS} attach ${id}
```

Comment:
I think we have used `docker logs --follow` previously here to get the output of the container process.